src/components/node-list/node-list-items.js

```diff
@@ -78,7 +78,6 @@ export const getFilteredNodes = createSelector(
   [(state) => state.nodes, (state) => state.searchValue],
   (nodes, searchValue) => {
     const filteredNodes = filterNodes(nodes, searchValue);
-
```

Comment:
I wonder why is there an empty row added here? It doesn't seem to relate to this PR and shouldn't be committed.